test/e2e/e2e-single-service/src/test/java/org/apache/skywalking/e2e/SampleVerificationITCase.java

```diff
@@ -111,7 +100,7 @@ public void verify() throws Exception {
         );
         assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
 
-        Thread.sleep(5000);
+        Thread.sleep(20000);
```

Comment:
I think we could do a re-try here. Do it every 20s, but we could try 5 times, in case of unexpected failure. What do you think?